<div style="padding-top: 20px; padding-left: 100px; padding-right: 110px; font-family:  Arial, Helvetica Neue, Verdana, Geneva, sans-serif !important; font-size:18px;">
	<div style="width:575px; margin-left: auto; margin-right: auto;">
		<p style="text-align:center"><img src="images/BAR-logo.png" style="width:200px;"></p>
		<h1 style="font-weight: normal; font-size: 50px; color: #99CC00; text-align:center">ePlant</h1>
		<p style="text-align:center; margin-bottom:40px;"><a href="http://www.waese.com" style="text-decoration:none;">Jamie Waese</a>, Jim Fan, Asher Pasha, Hans Yu and <a href="http://csb.utoronto.ca/faculty/provart-nicholas" style="text-decoration:none;">Nicholas Provart</a><br>Department of Cell & Systems Biology, University of Toronto</p>
	</div>

	<p>ePlant helps biologists visualize the natural connections between DNA sequences, molecular structure, protein-protein interactions, cell and tissue expression patterns, natural variation, and proteomic responses to various experimental conditions by combining several data visualization tools with a zoomable user interface. 
	</p>

	<!-- eplant visualization mockup -->
	<p style="text-align:center"><img src="images/ePlantVisualizationImage.jpg" style="width:400px;"></p>


	<p>ePlant connects to several publicly available web services to download the latest genome, proteome, metabolome, and tissue expression pattern data for any number of genes you may be interested in. It then displays that data with a set of visualization tools that are presented with a conceptual hierarchy from big to small. Zoom transitions between the views represent moving closer to- or further away from each level of analysis. Links between the different views help underscore connections between multiple levels of analysis.</p>


	<!-- biological levels image -->
	<p style="text-align:center"><img src="images/BiologicalLevels.png" style="width:500px;"></p>

	<p>You may be familiar with some of our visualization tools. We used <a href="http://cytoscape.github.io/cytoscape.js/">Cytoscape.js</a> for the protein interactions, <a href="http://wiki.jmol.org/index.php/Jmol_JavaScript_Object">JSmol</a> for the 3D molecular view, and <a href="http://jbrowse.org/">JBrowse</a> for the genome browser. The rest of ePlant was custom written with Javascript and runs locally in your browser.</p>


	<p>ePlant was built to be species agnostic. Please contact us if there is a particular species you would like to add. This project was funded with grants from Genome Canada and NSERC.</p>

	<p>This is a continuation of the original ePlant project by <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0015237&representation=PDF">Fucile et al. (2011)</a>.</p>


	<!-- logos at bottom of page -->
	<p>&nbsp;</p>
	<p style="text-align:center">
	<a href="http://www.csb.utoronto.ca/"><img src="images/csblogo.png" width="134" height="60" alt="Cell &amp; Systems Biology" longdesc="http://www.csb.utoronto.ca/" /></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
	<a href="http://www.cagef.utoronto.ca/"><img src="images/Cagef_logo.gif" width="87" height="57" /></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
	<a href="http://www.utoronto.ca"><img src="images/logoUofT.gif" width="180" height="65" alt="University of Toronto" longdesc="http://www.utoronto.ca/" /></a> 
	</p>

</div>